app/views/map/index.html.erb

```diff
@@ -1,23 +1,16 @@
-<!DOCTYPE html>
-<html>
-  <head>
-  	<title>HPI Map</title>
-	<meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+<!-- fullscreen -->
+<script src="https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/Leaflet.fullscreen.min.js"></script>
+<link rel="stylesheet" href="https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/leaflet.fullscreen.css" />
 
-	<%= stylesheet_link_tag 'map' %>
-	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet.locatecontrol@0.76.0/dist/L.Control.Locate.min.css" />
-  	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.3.1/leaflet.css" />
-	
-	
-	<%= content_tag :div, '', id: 'poi-data', data: { source: @points_of_interest } %>
-	<%= javascript_include_tag 'OutdoorMap/geometry' %>
-  </head>
+<%= stylesheet_link_tag 'map' %>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet.locatecontrol@0.76.0/dist/L.Control.Locate.min.css" />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.3.1/leaflet.css" />
 
-  <body>
-  	<div class="leaflet">
-  		<div id="map" style="width: 100%; height: 600px;"></div>
-		<%= javascript_include_tag 'OutdoorMap/map_build' %>
-		<%= render 'indoor' %>
-	</div>
-  </body>
-</html>
\ No newline at end of file
+<%= content_tag :div, '', id: 'poi-data', data: { source: @points_of_interest } %>
+<%= javascript_include_tag 'OutdoorMap/geometry' %>
+
+<div class="leaflet" style="width: 100%; height: 100%;">
+	<div id="map" style="width: 100%; height: 100%;"></div>
```

Comment:
Regarding my comment with the picture: I suppose this `div` not covering the whole page, excluding the navigation bar on the top, is what limits the "fullscreen"?

Reply:
No. The reason for the white borders around the map was some padding and margin specified in a higher level stylesheet. I can override this padding and margin though, the map indeed looks better without it.

Reply:
Looks great! I think now that it is "fullscreen mode" anyways, we could remove the fullscreen-handle, as that maybe clutters the interface a bit, especially on mobile where there is less space. What's your opinion on that?

Reply:
fullscreen has been removed in a05f1bef4147c5720b9f86db42956da2cf8ad797